app/aws/Federation.scala

```diff
@@ -42,7 +42,7 @@ object Federation {
   def duration(
       permission: Permission,
       requestedSeconds: Option[Duration] = None,
-      timezone: Option[DateTimeZone] = None
+      now: Option[ZonedDateTime] = None
```

Comment:
By passing in the current time, instead of time zone, we are able to test more easily because it's a static time.